client/blocks/importer/wordpress/import-everything/migration-error.tsx

```diff
@@ -0,0 +1,41 @@
+import { useChatWidget } from '@automattic/help-center/src/hooks';
+import { NextButton, SubTitle, Title } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+
+interface Props {
+	siteUrl: string;
+	resetMigration: () => void;
+}
+export default function MigrationError( props: Props ) {
+	const translate = useTranslate();
+	const { openChatWidget } = useChatWidget();
+	const { siteUrl, resetMigration } = props;
+
+	useEffect( () => {
+		openChatWidget( {
+			message: 'Import onboarding flow: migration failed',
```

Comment:
Maybe we can add the error code here, or the error message, so when HE folks pick this up they can save sometimes here. What do you think?